tests/test_config.py

```diff
@@ -161,7 +162,7 @@ def test_mutual_exclusion(self, mock_stderr):
         # for read_from
         keystring = 'this-is-a-private-key'
         strfile = tempfile.NamedTemporaryFile()
-        strfile.write('%s-written-to-file' % keystring)
+        strfile.write(('%s-written-to-file' % keystring).encode('ascii'))
```

Comment:
Is ASCII encoding (reversion) necessary? Are there current filesystems that don't support unicode/UTF-8?